server.js

```diff
@@ -0,0 +1,36 @@
+const express = require("express");
+const helmet = require("helmet");
+const cors = require("cors");
+const session = require("express-session");
+
+const server = express();
+
+server.use(
+  session({
+    name: "thisisaname", // default is connect.sid
+    secret: "eat my pants!",
```

Comment:
Typically we'll stick any secrets in a [.env](https://www.npmjs.com/package/dotenv) file; feel free to read up at that link (which I think I mentioned in our 1:1) and try to implement it if you're up for the challenge!